diff --git a/themes/twitter_bootstrap/includes/modules/form.inc b/themes/twitter_bootstrap/includes/modules/form.inc
index 9f23df1..c7be3fc 100755
--- a/themes/twitter_bootstrap/includes/modules/form.inc
+++ b/themes/twitter_bootstrap/includes/modules/form.inc
@@ -112,6 +112,10 @@ function twitter_bootstrap_form_element(&$variables) {
     case 'before':
     case 'invisible':
       $output .= ' ' . theme('form_element_label', $variables);
+      // Show field description help below field label and above field input
+      if ( !empty($element['#description']) ) {
+        $output .= '<p class="help-block">' . $element['#description'] . "</p>\n";
+      }
       $output .= '<div class="controls">';
       $output .= ' ' . $prefix . $element['#children'] . $suffix . "\n";
       break;
@@ -119,6 +123,10 @@ function twitter_bootstrap_form_element(&$variables) {
     case 'after':
       $output .= '<div class="controls">';
       $variables['#children'] = ' ' . $prefix . $element['#children'] . $suffix;
+      // Show field description help below field label and above field input
+      if ( !empty($element['#description']) ) {
+        $output .= '<p class="help-block">' . $element['#description'] . "</p>\n";
+      }
       $output .= ' ' . theme('form_element_label', $variables) . "\n";
       break;
 
@@ -130,10 +138,6 @@ function twitter_bootstrap_form_element(&$variables) {
       break;
   }
 
-  if ( !empty($element['#description']) ) {
-    $output .= '<p class="help-block">' . $element['#description'] . "</p>\n";
-  }
-
   $output .= "</div></div>\n";
 
   return $output;
